### DISCUSSION TITLE
MissionControl 0.6.3

### DISCUSSION THREAD

**Discussion by ndeadly** [ndeadly/MissionControl#399] [General]
<p align="left">
<img alt="GitHub Releases" src="https://img.shields.io/github/downloads/ndeadly/MissionControl/v0.6.3/total">
</p>

This is the twelfth official release of Mission Control

Just a quick release to support Atmosphère 1.2.5 and the new 13.2.0 firmware. Support for the Hyperkin Scout SNES controller was also added.

New users landing on this release page should first check out the readme on the [main project page](https://github.com/ndeadly/MissionControl#readme) for the official project documentation. There you can find installation and usage instructions along with an FAQ section that will answer most of your questions.

***Atmosphère >= 1.2.5 is required to run this release.***

### Important notice for users of Xbox One controllers
It has been confirmed that the new Xbox Insiders controller firmware update will convert your controller to use Bluetooth LE, which is not currently supported. Please do not update if you wish to continue using your controller with Mission Control

### Changelog:
- Updated to latest libstratosphere for 13.2.0 support
- Added support for Hyperkin Scout controller

<hr /><em>This discussion was created from the release <a href='https://github.com/ndeadly/MissionControl/releases/tag/v0.6.3'>MissionControl 0.6.3</a>.</em>

**Comment by zhangyuelong1988** [comment]
Hello, the left rocker of wiiu Pro handle will drift downward after using the new version. How can I solve this problem

**Reply by ndeadly** [reply]
Presumably you've updated from an older version from before controller deadzone values were emulated correctly. Try deleting the `sdmc:/config/MissionControl/controllers/` directory from your sd card, reconnecting your controller, and then completing the analog stick calibration under `System Settings->Controllers and Sensors->Calibrate Control Sticks`

**Comment by zhangyuelong1988** [comment]
Thank you for your reply. After deleting the file according to your requirements, connect the wiiu Pro handle. There is no option to correct the rocker in the system settings. I confirmed that my handle is good. Then I connected it with another fake wiiu Pro handle, and the rocker did not drift. Under the previous 13.0 system, using the 0.6.0 version, there was no rocker drift.

**Reply by ndeadly** [reply]
> There is no option to correct the rocker in the system settings.

The option is there, maybe it's named differently in your system language.

![2022010615420200-57B4628D2267231D57E0FC1078C0596D](https://user-images.githubusercontent.com/24677491/148401399-bb95b184-4ba1-4750-8d7e-44f5d6c67db2.jpg)
![2022010615421700-57B4628D2267231D57E0FC1078C0596D](https://user-images.githubusercontent.com/24677491/148401408-96a9be5b-5151-4ad2-99e5-dc4474fd7f41.jpg)
![2022010615424900-57B4628D2267231D57E0FC1078C0596D](https://user-images.githubusercontent.com/24677491/148401410-52416872-8de2-4a9a-aef7-3971aad3b46f.jpg)

> Under the previous 13.0 system, using the 0.6.0 version, there was no rocker drift.

This makes sense, as the fix I mentioned was applied in v0.6.1. The improved accuracy can cause a small analog stick center offset that might have always been present to be amplified, resulting in drift. Recalibrating should resolve it.

From the 0.6.1 [release notes](https://github.com/ndeadly/MissionControl/releases/tag/v0.6.1):

> Valid analog stick deadzone parameters are now stored in virtual SPI flash file for each controller. This resolves an issue reported by several users where some controllers were experiencing analog stick drift even after calibration due to incorrect deadzone parameters being applied. Please delete the controller entries under sdmc:/config/MissionControl/controllers/ so that these files can be recreated with the correct parameters. Note that you will need to redo any user calibrations after deleting these entries.

**Reply by zhangyuelong1988** [reply]
If the wiiu Pro handle is connected, the calibrate control sticks and key test options will disappear, so the rocker cannot be calibrated. If the switch Pro handle and wiiu Pro handle are connected, the calibrate control sticks and key test options will also disappear.

**Reply by ndeadly** [reply]
Which firmware version are you on? Can you create a video or something demonstrating the behaviour you describe? I have never seen these options disappearing before.

**Reply by ndeadly** [reply]
Oooh, I think I might know what you're talking about. There's a weird behaviour that I've noticed only on some controllers like the Wii U Pro where you can't scroll down the controller settings menu past a certain point using the d-pad for some reason. I can't think of any good explanation for this, and especially why it would only affect certain controllers. Using the analog stick instead will allow you to scroll down to the lower items. Could this be the issue?

**Reply by zhangyuelong1988** [reply]
Yes, that's the problem. At first, I thought the option disappeared. After your reminder, I found that I couldn't scroll down to the calibrate control sticks option, so the rocker drift can't be adjusted through the system.

**Reply by ndeadly** [reply]
As I said, you should be able to scroll down to the option using the left or right analog stick, the joycons, or even the touchscreen. You probably don't even need the controller you want to calibrate connected until you're already in the calibration menu.

**Reply by zhangyuelong1988** [reply]
Thank you very much. The problem that has bothered me for a long time has been solved,i cannot scroll down with the arrow keys,It's the problem that the cross key can't scroll down. Now I can calibrate the rocker. After calibration, the left rocker of wiiu Pro handle doesn't drift. My English is poor. You can still help me. You are my idol.
